pages/common/ftp.md

```diff
@@ -0,0 +1,27 @@
+# ftp
+
+> Internet File Transfer Protocol
+
+- Connect to a FTP server:
+
+`ftp {{ftp.example.com}}`
+
+- Switch to binary transfer mode (graphics, compressed files etc):
+
+`binary`
+
+- Turn off interactive prompting for multiple file transfers:
```

Comment:
It would be clearer to explain what the prompt is for. One could make reasonable assumptions that this description doesn't clarify, e.g. it could be for confirmation or overwriting, and could activate only if glob expressions are used (not an explicit list of files), etc.